blaze/utils.py

```diff
@@ -193,6 +193,17 @@ def json_dumps(f):
     return {'__!builtin_function': f.__name__}
 
 
+# conscturct a list of all numpy functions and their types
+
+@dispatch(Callable)
+def json_dumps(f):
+    if f.__module__.startswith('numpy') or f.__module__.startswith('pandas'):
+        fcn = ".".join([f.__module__, f.__name__])
+    else:
+        raise NotImplementedError("Only supports numpy or pandas functions")
+    return {'__!numpy_pandas_function': fcn}
+
```

Comment:
Added `json_dumps` for pandas/numpy functions used in `map`. Only accepts pandas/numpy functions - the builtin functions dispatch on a separate function.

Is this a good way to ensure we only handle pandas/numpy functions? The builtin functions are handled by a different json_dumps that triggers for `types.BuiltinFunctionType`

@kwmsmith